jf_agent/jf_jira/jira_download.py

```diff
@@ -111,65 +94,17 @@ def run_jira_download(config: ValidatedConfig, ingest_config: IngestionConfig) -
 @logging_helper.log_entry_exit(logger)
 def detect_and_repair_custom_fields(
     ingest_config: IngestionConfig, submit_issues_for_repair: Optional[bool] = True
-) -> set[str]:
+) -> None:
     jira_connection = get_jira_connection_from_jf_ingest(ingest_config.jira_config)
     deployment_type = jira_connection.server_info()['deploymentType']
 
     # NOTE: This is temporary! For initial roll out, we want to target only Cloud Instances
     # After stable roll out, we'd like to target Server instances as well
     if deployment_type != 'Cloud':
         logger.info('Skipping detection and repair of custom fields, deployment type is not Cloud')
-        return set()
-
-    kwargs = dict()
-
-    logger.info("Beginning detection of custom field mismatches")
-    jcfv_update_payload: JCFVUpdateFullPayload = identify_custom_field_mismatches(
-        ingest_config, **kwargs
-    )
-
-    # Logging
-    update_counts = defaultdict(int)
-    for update in (
-        jcfv_update_payload.out_of_sync_jcfv
-        + jcfv_update_payload.missing_from_db_jcfv
-        + jcfv_update_payload.missing_from_jira_jcfv
-    ):
-        update_counts[f'{update.value_old} -> {update.value_new}'] += 1
-    for value_change, update_count in sorted(
-        update_counts.items(), key=lambda x: x[1], reverse=True
-    ):
-        if value_change.startswith('None -> '):
-            prefix = 'INSERT'
-        elif value_change.endswith(' -> None'):
-            prefix = 'DELETE'
-        else:
-            prefix = 'UPDATE'
-        logger.info(f'{update_count: >6d} | {prefix} | {value_change }')
-
-    if submit_issues_for_repair:
-        logger.info("Submitting list of issue IDs to repair to Jellyfish")
-
-        missing_db_ids = [jcfv.jira_issue_id for jcfv in jcfv_update_payload.missing_from_db_jcfv]
-        missing_jira_ids = [
-            jcfv.jira_issue_id for jcfv in jcfv_update_payload.missing_from_jira_jcfv
-        ]
-        missing_out_of_sync_ids = [
-            jcfv.jira_issue_id for jcfv in jcfv_update_payload.out_of_sync_jcfv
-        ]
-
-        all_ids = set(missing_db_ids + missing_jira_ids + missing_out_of_sync_ids)
-        logger.info(
-            f'Submitting {len(all_ids)} issue IDs to Jellyfish that were marked for redownload'
-        )
-
-        # TODO: SUBMIT ALL_IDS TO JELLYFISH TO MARK THEM FOR REDOWNLOAD IN OUR SYSTEM
-
-        logger.info(
-            f'Done submitting issues for repair. {len(all_ids)} issues were marked for redownload'
-        )
```

Comment:
All of the above logging is now present in `jf_ingest` version `0.0.138`, and is redundant here.